presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -6731,6 +6733,26 @@ public void testTimeLiterals()
     }
 
     @Test
+    public void testArrayShuffle()
+            throws Exception
+    {
+        int testCount = 23;
```

Comment:
Inline this variable. It's only used in one place. Also, 23 is a weird number. I'd pick something more regular, like 10 or 100.